src/Contracts/ManagerContract.php

```diff
@@ -9,5 +9,5 @@
 
 interface ManagerContract
 {
-    public function browse(Command $command, Closure $callback): void;
```

Comment:
This would force a major version, because is a breaking change. There is no other way of adding this feature without updating the interface?

Reply:
I may be better don't provide this feature.